services/galley/schema/src/Main.hs

```diff
@@ -63,9 +64,10 @@ main = do
       V32.migration,
       V33.migration,
       V34.migration,
-      V35.migration, -- This should safe to do now
+      V35.migration,
       V36.migration,
-      V37.migration
+      V37.migration,
+      V39.migration
```

Comment:
Note that here I used V39 as I'd like https://github.com/wireapp/wire-server/pull/1081 to get merged first and then can readjust this in an easier way.